### PR TITLE
chore(macros/AddonSidebar*): update zh-CN translation

### PR DESCRIPTION
**Pull request by skyclouds2001**
## Summary

update the translation of zh-cn for {{AddonSidebar}} and {{AddonSidebarMain}}

### Solution

---

### Before

image

image

### After

image

image

---

## How did you test this change?

watch the develop result

---

/cc @yin1999 @JasonLamv-t @jasonren0403 

part of the https://github.com/mdn/translated-content/issues/13881

### CODE REVIEW

File: kumascript/macros/AddonSidebar.ejs [hunk 1]
```diff
@@ -85,8 +85,8 @@ var text = mdn.localStringMap({
     'extension_workshop_manage': 'Manage',
     'extension_workshop_enterprise': 'Enterprise',
     'Channels': '渠道',
-    'Blog': 'Add-ons博客',
-    'Forums': 'Add-ons论坛',
+    'Blog': 'Add-on 博客',
+    'Forums': 'Add-on 论坛',
     'Chat': 'Add-ons chat',
```
Comment (yin1999) [inline kumascript/macros/AddonSidebar.ejs:90]: Add-ons => 附加组件

https://transvision.mozfr.org/?recherche=Add-ons&repo=all_projects&sourcelocale=en-US&locale=zh-CN&search_type=strings

**Comment by caugner** [comment]
@skyclouds2001 Thank you for your PR. Please make sure to fill out the PR description, including screenshots, to make it easier for reviewers. 🙏

File: kumascript/macros/AddonSidebar.ejs [hunk 2]
```diff
@@ -77,17 +77,17 @@ var text = mdn.localStringMap({
     'WebExtensions#Concepts': '概念',
     'WebExtensions#User_Interface': '用户界面',
     'WebExtensions#How_to': '怎么做',
-    'WebExtensions#API': 'JavaScript APIs',
-    'WebExtensions#manifest.json': 'Manifest keys',
+    'WebExtensions#API': 'JavaScript API',
+    'WebExtensions#manifest.json': 'Manifest key',
```
Comment (yin1999) [inline kumascript/macros/AddonSidebar.ejs:81]:
```suggestion
    'WebExtensions#manifest.json': 'Manifest 键',
```

**Review by yin1999 — APPROVED** [review]
Only one nit. Otherwise LGTM. Thank you @skyclouds2001

**Review by yin1999 — APPROVED** [review]

**Review by caugner — APPROVED** [review]

**Pull request merged**